avocado/utils/kernel.py

```diff
@@ -18,8 +18,9 @@
 import logging
 import tempfile
 from distutils.version import LooseVersion  # pylint: disable=E0611
+from avocado.utils import distro
```

Comment:
This is actually flagged by the "boundary check":

https://travis-ci.org/avocado-framework/avocado/jobs/355619977#L1518

Please use:

```python
from . import distro
```

Reply:
Thank you; making the correction.